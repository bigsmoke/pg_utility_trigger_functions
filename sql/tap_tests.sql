begin;

create schema if not exists ext;

set search_path to ext;

create extension pg_utility_trigger_functions
    with schema ext cascade;

call test__update_updated_at();
call test__no_delete();
call test__nullify_columns();
call test__copy_fields_from_foreign_table();
call test__fallback_to_fields_from_foreign_table();
call test__set_installed_extension_version_from_name();
call test__coalesce_sibling_fields();

rollback;
