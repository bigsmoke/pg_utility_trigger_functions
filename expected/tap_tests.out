begin;
create schema if not exists ext;
set search_path to ext;
create extension pg_utility_trigger_functions
    with schema ext cascade;
NOTICE:  installing required extension "hstore"
call test__update_updated_at();
call test__no_delete();
call test__nullify_columns();
call test__copy_fields_from_foreign_table();
call test__fallback_to_fields_from_foreign_table();
rollback;
